docs/howto/create-a-release.rst

```diff
@@ -102,6 +102,115 @@ Upload build to PyPI
 
 Check PyPI for your upload.
 
+Upload to Conda
+===============
+
+NOTE: the GitHub and PYPI releases are prerequisites for releasing to
+Conda Forge!
+
+The Conda Forge release process is mostly automated by bots. These bots
+are set up during the initial project bootstrap, which has been done for
+all the projects that are part of Cylc 8.
+
+As there are inter-dependencies amongst the Cylc 8 parts, you should
+know the dependency tree, and start by the leaf nodes (i.e. a module
+A may have multiple dependants, but no dependency to other modules).
+
+For example, Cylc UI Server depends on Cylc Flow. So unless you are
+releasing only Cylc UI Server, you should release Cylc Flow first.
+
+* On GitHub, navigate to the project repository on GitHub, e.g.
+
+- https://github.com/conda-forge/metomi-isodatetime-feedstock
+- https://github.com/conda-forge/cylc-uiserver-feedstock
+- https://github.com/conda-forge/cylc-ui-feedstock
+- https://github.com/conda-forge/cylc-flow-feedstock
+- https://github.com/conda-forge/cylc-feedstock
+
+On each of these repositories, the release process should be pretty
+much the same. Your work will be mainly (if not exclusive) on
+the ``recipe/meta.yaml`` file.
+
+This is a possible order of action:
+
+- Check open pull requests—There are bots that alert about dependencies
+(N.B. conda packages do not use PYPI for dependencies, they all use
+other conda packages as dependencies!.
+- Check if you need to update the version in the Jinja2 variable right
+at the top of the file.
+- If you updated the version, set the build number back to 0.
+- If you did not update the version, increase the build number.
+- If the conda package contains a source URL, check that it is evaluated
+correctly after the version is replaced, and update the ``sha256`` value.
+- Check requirements (we may have changed setup.py, or package.json, etc.
+ensure we are up to date).
+- Ensure the test commands are still valid
```

Comment:
What marks them out as valid?
What should they do?

Reply:
Added the following

>- Ensure the test commands are still valid (i.e. they can be executed successfully)

As the commands are YAML strings that may represent Python commands, Python imports, shell commands, etc. So a developer releasing Cylc UI Server should see a test command such as 

```
test:
  imports:
    - cylc.uiserver
```

and could either test it with `python -c 'import cylc.uiserver'` after the package was installed locally, or simply confirm that we have not moved/renamed packages (I would do the latter).

Reply:
That would have been my guess without futher clarification, so I'm happy w' that.